### PR TITLE
Update 'Getting started for engineers' content

### PR DESCRIPTION
**Pull request by alex-ju**
### :pushpin: Summary

First pass at 'Getting started for engineers' content

### :hammer_and_wrench: Detailed description

Most of this content is based on current instructions written in GitHub or dedicated pages in the dummy website.

I tried keeping it to a minimum to avoid overwhelming consumers or putting them off by a wall of content. We can always add more scenarios later (for example alternative setup instructions, such as npm or postcss) if we find patterns across products.

### :link: External links

Jira ticket: [HDS-1069](https://hashicorp.atlassian.net/browse/HDS-1069)

***

### 👀 Reviewer's checklist:

- [ ] +1 Percy if applicable
- [ ] Confirm that PR has a changelog update via [Changesets](https://github.com/changesets/changesets) if needed

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

### CODE REVIEW

**Review by MelSumner — APPROVED** [review]
👍

File: website/docs/getting-started/for-engineers.md [hunk 1]
```diff
@@ -1,9 +1,110 @@
 ---
 title: For engineers
-description: This is the (missing) long description of the component, that will come from the frontmatter attributes
 order: 102
 ---
 
-## Lorem ipsum dolor
+The Design System team maintains design tokens, icons, and components in a [monorepo](https://github.com/hashicorp/design-system) and publishes them regularly to npm.
```
Comment (MelSumner) [inline website/docs/getting-started/for-engineers.md:6]:
```suggestion
The Design Systems team maintains design tokens, icons, and components in a [monorepo](https://github.com/hashicorp/design-system) and publishes them regularly to npm.
```

File: website/docs/getting-started/for-engineers.md [hunk 2]
```diff
@@ -1,9 +1,110 @@
...
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Diam diam mi etiam mollis tortor vestibulum. Felis, arcu amet, nibh eget eget etiam orci aliquet. Varius facilisis magna faucibus commodo iaculis faucibus aliquet.
\ No newline at end of file
+## Components
+
+We provide components as an [Ember](https://emberjs.com/) addon with associated styles written in [Sass](https://sass-lang.com/). By installing the components you also have access to the design tokens and CSS helper classes.
```
Comment (MelSumner) [inline website/docs/getting-started/for-engineers.md:10]:
```suggestion
We provide components as an [Ember](https://emberjs.com/) addon with associated styles written in [Sass](https://sass-lang.com/). By installing the components, you also have access to the design tokens and CSS helper classes.
```

File: website/docs/getting-started/for-engineers.md [hunk 3]
```diff
@@ -1,9 +1,110 @@
...
+```
+
+This package can be consumed in React applications via direct import of the SVG file or as a standalone React/SVG icon component.
+
+For more details, examples and guidelines read [the complete icons documentation](/foundations/icons/).
+
+
+## Tokens
+
+If the Ember components are not an option for your project you can use design tokens to keep in sync with the styles we provide.
```
Comment (MelSumner) [inline website/docs/getting-started/for-engineers.md:81]:
```suggestion
If the Ember components are not an option for your project, you can still use the design tokens to keep in sync with the styles we provide.
```

File: website/docs/getting-started/for-engineers.md [hunk 4]
```diff
@@ -1,9 +1,110 @@
...
+  includePaths: [
+    './node_modules/@hashicorp/design-system-tokens/dist/products/css',
+  ],
+},
+```
+
+Add the following line to the main Sass file in your application (for example, in `app.scss`):
+
+```scss
+@import '@hashicorp/design-system-components';
```
Comment (KristinLBradley) [inline website/docs/getting-started/for-engineers.md:42]: Super nit: We use double quotes within Scss code in HDS styles. (I know consumers may have different standards though.)

Comment (alex-ju) [reply website/docs/getting-started/for-engineers.md:42]: Happy to update to double 👍

**Review by KristinLBradley — APPROVED** [review]

File: website/docs/getting-started/for-engineers.md (hunk 4 above)
Comment (KristinLBradley) [reply website/docs/getting-started/for-engineers.md]: Thanks!

**Pull request merged**